### ISSUE TITLE
open-android-backup-companion.apk Problem

### ISSUE THREAD

**Issue by Fossa59** [mrrfv/open-android-backup#109]
## Device info

Manufacturer:Xiaomi
Android version:13  MIUI 14
Custom ROM/rooted? No
Model:Redmi note 11 pro 5g

## System info

Available disk space:1 TB
Operating system:Opensuse Leap 15.5

---

**Describe the issue below.**
Using Open_android_Backup_v1.0.16_Bundle and it works perfectly only when i install open-android-backup-companion.apk and start it by hand. Running the backup.sh without doing this before the script runs up to the point to backup the contacts, sms and call logs and then aborts.
"Exporting contacts (as vCard), call logs as well as SMS messages (as CSV).
Errors occurred while backing up . - this file (or multiple files) might've been ignored.
No contacts found on device - ignoring."
open-android-backup-temp is not created on the phone. When installed by hand and started only this way the questions for allowing acces to contacts sms and phone calls appears. Then starting rhe script -- every thing is working perfect until the end.
Thank you for the work! I hope you understand what i wrote, because its written by human intelligence and not translated by google.

**Comment by mrrfv** [comment]
Is the app even installed by the script at all? Or it simply has problems with getting the correct permissions?

**Comment by Fossa59** [comment]
Yes app is installed and pretty soon uninstalled without doing the copy of sms and others. And as i pasted above: "No contacts found on device - ignoring."

**Comment by wicadmin** [comment]
Having same type of symptoms

**Comment by Klaasstap** [comment]
The Companion App can´t find my contacts either.
